src/styles/styles.js

```diff
@@ -3281,7 +3286,7 @@ const styles = {
     newKansasLarge: {
         ...headlineFont,
         fontSize: variables.fontSizeXLarge,
-        lineHeight: 27,
+        lineHeight: variables.lineHeightXXLarge,
```

Comment:
Changing the line height to scale by the device font size while the font size is static, causing an [emoji to be clipped on iOS when the device font size is reduced](https://github.com/Expensify/App/issues/74523).